trace/trace.proto

```diff
@@ -146,7 +139,7 @@ message Span {
       Annotation annotation = 2;
 
       // An event describing an RPC message sent/received on the network.
-      NetworkEvent network_event = 3;
+      MessageEvent network_event = 3;
```

Comment:
Ps you still want the old field name?

Reply:
Several of the comments still mention NetworkEvent, too.